public/main.js

```diff
@@ -0,0 +1,13 @@
+fetch('/getcities')
+.then(response => {
+    return response.json();
+})
+.then((cities)=>{
+    var list = document.getElementById("select_city");
```

Comment:
You could use `let` instead of `var` since you are using fetch which is ES6

Reply:
we will do it